etc/kayobe/seed.yml

```diff
@@ -106,8 +106,11 @@ seed_pulp_container:
     image: pulp/pulp
     pre: "{{ kayobe_config_path }}/containers/pulp/pre.yml"
     post: "{{ kayobe_config_path }}/containers/pulp/post.yml"
-    tag: "3.16"
+    tag: "3.21"
     network_mode: host
+    # Override deploy_containers_defaults.init == true to ensure
+    # s6-overlay-suexec starts as pid 1
+    init: false
```

Comment:
Does it work with this?

Reply:
yes it does. tested with latest 3.21 image from docker hub (1c48e11a0e64)